code/modules/cargo/bounties/chef.dm

```diff
@@ -245,16 +254,14 @@
 	reward_high = 5500
 	required_count = 3
 	random_count = 1
-	wanted_types = list(/obj/item/reagent_containers/food/snacks/adhomian_sausage,
-			/obj/item/reagent_containers/food/snacks/fermented_worm,
-			/obj/item/reagent_containers/food/snacks/hardbread,
-			/obj/item/reagent_containers/food/snacks/lardwich,
-			/obj/item/reagent_containers/food/snacks/nomadskewer,
-			/obj/item/reagent_containers/food/snacks/soup/earthenroot,
-			/obj/item/reagent_containers/food/snacks/spicy_clams,
-			/obj/item/reagent_containers/food/snacks/stew/tajaran,
-			/obj/item/reagent_containers/food/snacks/tajaran_bread,
-			/obj/item/reagent_containers/food/snacks/tajcandy)
+	wanted_types = list(/obj/item/reagent_containers/food/snacks)
+
+/datum/bounty/item/chef/tajara/applies_to(var/obj/item/reagent_containers/food/snacks/O)
+	if(!..())
+		return FALSE
+	if(O.icon == 'icons/obj/item/reagent_containers/food/cultural/tajara.dmi')
+		return TRUE
```

Comment:
checking the icon isn't an acceptable way to do this - icon files are prone to being changed and generally should never be used to check the state or type of an object